ArduCopter/mode.h

```diff
@@ -780,6 +782,7 @@ class ModeGuided : public Mode {
 
     uint32_t wp_distance() const override;
     int32_t wp_bearing() const override;
+    float crosstrack_error() const override { return wp_nav->crosstrack_error();}
```

Comment:
This needs to depend on `guided_mode == Guided_WP`, other modes don't have a crosstrack error.